bin/clover/src/spec/sockets.ts

```diff
@@ -47,6 +47,14 @@ export function createInputSocketFromProp(
   return socket;
 }
 
+export function getSocketOnVariant(
```

Comment:
If we're adding something here anyway, might be better to add getOrCreateOutputSocketFromProp() instead, to match getOrCreateInputSocketFromProp(). But just a nit.